create_instances_module.rb

```diff
@@ -30,6 +31,26 @@ def add_book
     @books << complete_book
   end
 
+  def add_game
+    p "What's the publish date? [year]: "
+    game_p_year = gets.chomp.to_i
```

Comment:
Great job my friend! these method is looking good.
Please modify it so it doesn't ask for a year but for a full date (ex: 12-12-2020)
We need to use the date object so we can save it to the file
- remove the `[year]` part or the prompt in line 35
- parse the user's input into a date object (ex: `Date.parse(gets.chomp)`)